speech/cloud-client/transcribe_stream.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# author: Yogesh Garg
+# email: yogeshg91@gmail.com
+# github: yogeshg
+
+"""Google Cloud Speech API sample application for transcribing audio streams
+
+Example usage:
+    python transcribe_stream resources/audio.raw
+ 
+sources:
+ * https://googlecloudplatform.github.io/google-cloud-python/stable/speech-usage.html#streaming-recognition
+
+"""
+
+import logging
```

Comment:
I'm not 100% sure what this does but I imagine you would want to follow the style [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/3ae5f4691de8d03d87a899fda0b1ebce522371e0/speech/cloud-client/transcribe.py#L25-L28) with the `# [START import_libraries]`.